arches/app/views/resource.py

```diff
@@ -596,7 +596,7 @@ def get(self, request, resourceid=None, view_template="views/resource/edit-log.h
                 resource_type=graph_name,
                 resource_description=resource.displaydescription(),
                 iconclass=resource_instance.graph.iconclass,
-                edits=JSONSerializer().serialize(permitted_edits),
+                edits=localize_serialized_object(JSONSerializer().serialize(permitted_edits)),
```

Comment:
It looks like `permitted_edits` is getting serialized here, then deserialized at the start of `localize_serialized_object` and then re-serialized at the end of `localize_serialized_object`. Could permitted_edits be localized here first and then serialized? Basically flip the order like this:
`JSONSerializer().serialize(localize_object(permitted_edits)),`

Reply:
ya good advice